pkg/k8s/types/types.go

```diff
@@ -49,8 +50,9 @@ type SlimCNP struct {
 }
 
 type ContainerPort struct {
-	Protocol      string
-	ContainerPort int32
+	Name          string `json:"name,omitempty"`
+	Protocol      string `json:"protocol,omitempty"`
+	ContainerPort int32  `json:"containerPort"`
```

Comment:
why json tags?

Reply:
Recall adding them due to empty name popping up in (unit) tests, but can't be sure, will test again without.

Reply:
```
l4_test.go:185:
    c.Assert(model.Egress[i].Rule, Equals, expectedEgress[i])
... obtained string = "" +
...     "{\n" +
...     "  \"port\": 8080,\n" +
...     "  \"protocol\": \"TCP\"\n" +
...     "}"
... expected string = "" +
...     "{\n" +
...     "  \"port\": 8080,\n" +
...     "  \"port-name\": \"\",\n" +
...     "  \"protocol\": \"TCP\"\n" +
...     "}"
... String difference:
...     []string[4] != []string[5]
```

Reply:
I can't find any reason why the json tags should be here, so taking them away :-)